pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -395,18 +418,30 @@ let
     fetchAttrs = {
       sha256 = {
       x86_64-linux = if cudaSupport
-        then "sha256-lURiR0Ra4kynDXyfuONG+A7CpxnAsfKzIdFTExKzp1o="
-        else "sha256-lDvRgj+UlaneRGZOO9UVCb6uyxcbRJfUhABf/sgKPi0=";
-      aarch64-linux = "sha256-z2d45fqHz5HW+qkv3fR9hMg3sEwUzJfxF54vng85bHk=";
-      x86_64-darwin = "sha256-AAvuz8o6ZRkaSYMgaep74lDDQcxOupDCX4vRaK/jnCU=";
-      aarch64-darwin = "sha256-kexRSvfQqb92ZRuUqAO070RnUUBidAqghiA7Y8do9vc=";
+        then "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+        else "sha256-bMo7tUxeAYMFXcrgv3WDlvxaeiP0jWo0rn8yg78mXDU=";
+      aarch64-linux = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
+      x86_64-darwin = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
+      aarch64-darwin = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```

Comment:
leaving a note that we'll need to fix the rest of these before merging ofc!

Reply:
Sure. I'm open to ideas how to efficiently acquire the deps hashes for all the platforms. I was able to grab one from ofborg once (https://github.com/NixOS/nixpkgs/pull/255998/commits/063c4e881af3c41fea58161fb6159d45a6b121c0), but that one is invalidated through moving to more syslibs by now, and since then, ofborg always fails due to exceeding the 3600s limit before even reaching `tensorflow`.